db/scrub_access_limited.sql

```diff
@@ -0,0 +1,24 @@
+-- This script is to be executed by https://github.com/alphagov/env-sync-and-backup
+-- (private repo) as part of the data sync from production to integration.
+-- It is used to remove any access limited content from Content Publisher at
+-- the point of copying to integration.
+-- It is kept in this repo to be tested with the application, whenever changes
+-- are made it should be re-copied to the env-sync-and-backup repository.
+
+-- Remove current status of any access limited editions
+UPDATE editions
+SET current = false
+WHERE access_limit_id IS NOT NULL;
+
+-- Set the live editions to current for any docs with an access limited draft
+UPDATE editions
```

Comment:
In the case that a document only has one edition, which is access limited, I assume we'll end up with a document that has no editions.

How does content publisher deal with this?  Will it cause exceptions to be raised in integration (and when the data is re-used locally) if someone attempts to view this document?

Reply:
That scenario is ok in Content Publisher. It's quite similar to when someone discards the first draft of a document and we end up with a document without a current or live edition. Our queries are based on an edition that joins on document rather than the other way round.

Thanks for reviewing @brucebolt 👍